templates/resources/load_balancers.tfvars.j2

```diff
@@ -0,0 +1,110 @@
+load_balancers = {
+{% for key, lbs in resources[tfstate_resource].resources[subscription_key].load_balancers.items() %}
+  {{ key }} = {
+    name                      = "{{ lbs.name }}"
+    resource_group_key        = "{{ lbs.resource_group_key }}"
+    sku                       = "{{ lbs.sku }}"
+{% if lbs.backend_address_pool_name is defined %}
+    backend_address_pool_name               = "{{ lbs.backend_address_pool_name }}"
+{% endif %}
+{% if lbs.frontend_ip_configurations is defined %}
+    frontend_ip_configurations = {
+{% for frontend_key, config in lbs.frontend_ip_configurations.items() %}
+      {{ frontend_key }} = {
+        name    = "{{config.name}}"
+{% if config.private_ip_address is defined %}
+        private_ip_address = "{{config.private_ip_address}}"
+{% endif %}
+{% if config.private_ip_address_allocation is defined %}
+        private_ip_address_allocation = "{{config.private_ip_address_allocation}}"
+{% endif %}
+{% if config.private_ip_address_version is defined %}
+        private_ip_address_version = "{{config.private_ip_address_version}}"
+{% endif %}
+{% if config.public_ip_address_key is defined %}
+        public_ip_address_key = "{{config.public_ip_address_key}}"
+{% endif %}
+{% if config.public_ip_prefix_id is defined %}
+        public_ip_prefix_id  = "{{config.public_ip_prefix_id}}"
+{% endif %}
+{% if config.vnet_key is defined %}
+        vnet_key    = "{{config.vnet_key}}"
+{% endif %}
+{% if config.subnet_key is defined %}
+        subnet_key    = "{{config.subnet_key}}"
+{% endif %}
+{% if config.lz_key is defined %}
+        lz_key     = "{{ config.lz_key }}"
+{% endif %}
+{% if config.zones is defined %}
+        zones     = "{{ config.zones }}"
```

Comment:
zones is a list. So you need to adjust the code to get it back to a terraform HCL list

```
{% if disk_value.zones is defined %}
          zones = {{ disk_value.zones | map('string') | replace('None','[]') | replace('\'','\"') }}
{% endif %}
```